server/api/commands.go

```diff
@@ -53,20 +44,29 @@ func LoadTest(w stdio.Writer, state State, waitForCancel func(), throttle string
 		io.ReplyEither(w, err)
 		return result
 	}
-	var targets t.Targets = []t.Target{
-		{"POST", url, reqBody, conc, float64(qps)},
-	}
-	work := targets.GetWork(state.ApmServer().Url(), &t.Config{
-		MaxRequests: math.MaxInt32,
-		// should match the one in apm-server
-		RequestTimeout:     30,
+
+	cfg := &target.Config{
+		Concurrent:     conc,
```

Comment:
I like the renaming from `conc`/`concurrent` to `numAgents`, wdyt about applying it everywhere?